jck/jck.mk

```diff
@@ -68,8 +68,13 @@ endif
 OTHER_OPTS=
 # if JDK_IMPL is openj9 or ibm
 ifneq ($(filter openj9 ibm, $(JDK_IMPL)),)
- OTHER_OPTS=-Xtrace:maximal=all{level2}
- export CONFIG_ALT_PATH:=$(JCK_ROOT)$(D)config$(D)default
```

Comment:
Please put back the deleted line 72. It should not be removed as part of this PR.

Reply:
Hi @Mesbah-Alam, I just revoked the deleted line, please review when you are free!